--- original/Documentation/meson.build   2023-05-07 13:18:36.536057257 +0200
+++ kgwportage/Documentation/meson.build_docdir    2023-05-07 17:33:40.825574091 +0200
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: CC0-1.0
 
-doc_install_dir = get_option('datadir') / 'doc' / 'libcamera-@0@'.format(libcamera_version)
+doc_install_dir = get_option('datadir') / 'doc' / 'libcamera-raspberrypi-@0@'.format(libcamera_version)
 
 #
 # Doxygen
