--- origsource/CMakeLists.txt	2026-03-01 10:50:23.670683693 +0100
+++ kgwportage/CMakeLists.txt	2026-03-01 13:48:53.959591631 +0100
@@ -766,7 +766,7 @@
 
 	   set (${objectName}_MOCS
 	        ${${objectName}_MOCS}
-	        ./sources/devices/rtlsdr-handler/rtl-dongleselect.h
+		# ./sources/devices/rtlsdr-handler/rtl-dongleselect.h
 	        ./sources/devices/rtlsdr-handler/rtlsdr-handler.h
 	   )
 
@@ -777,7 +777,7 @@
 	   set (${objectName}_HDRS
 	        ${${objectName}_HDRS}
 	        ./sources/devices/rtlsdr-handler/rtlsdr-handler.h
-	        ./sources/devices/rtlsdr-handler/rtl-dongleselect.h
+		# ./sources/devices/rtlsdr-handler/rtl-dongleselect.h
 	        ./sources/devices/rtlsdr-handler/rtl-sdr.h
 	        ./sources/devices/rtlsdr-handler/rtl-sdr_export.h
 	   )
@@ -785,7 +785,7 @@
 	   set (${objectName}_SRCS
 	        ${${objectName}_SRCS}
 	        ./sources/devices/rtlsdr-handler/rtlsdr-handler.cpp
-	        ./sources/devices/rtlsdr-handler/rtl-dongleselect.cpp
+		# ./sources/devices/rtlsdr-handler/rtl-dongleselect.cpp
 	   )
 
 	   add_definitions (-DHAVE_RTLSDR)
